scripts/js_texts/extracted_strings_app.js

```diff
@@ -3,9 +3,9 @@ module.exports = [
     'AM',
     'Account',
     'Account Authenticated',
+    'Account Authentication',
```

Comment:
should the original string change to `Account authentication` instead?

Reply:
Yep, you're right. 👍